score/ipython/Score_Dissolve_Script.ipynb

```diff
@@ -0,0 +1,454 @@
+    },
+    "id": "_C6vaR9HQeLa",
+    "outputId": "fab3bc7f-e716-431e-bc76-bd26289ea4a4"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "master_df.head(2)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "oMoubjqCQiw5",
+    "outputId": "6195ffbc-6275-40c6-bb6a-e0a6bd1e71f0"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_df = pd.read_csv(CEJST_DATA_PATH, dtype={\"GEOID10\": \"object\"})"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "bAMmGSgzVml0"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged = master_df.merge(usa_df, on=\"GEOID10\", how=\"left\")"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "U7M7dExdV2Vh"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged.head(2)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "Sr25DUkxWVhg",
+    "outputId": "1e804075-0f7d-4174-82d7-e21b8519c8bf"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged_compressed = gpd.GeoDataFrame(usa_merged, crs=\"EPSG:4326\")"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "ANMlAB8Qmtu8",
+    "outputId": "44934741-90a9-4664-fab5-2c39b348d2be"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged_compressed.to_file(\"usa_merged.geojson\", driver=\"GeoJSON\")"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "PBPD9LQctvPJ"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_simplified = usa_merged[\n",
+    "        [\"GEOID10\", \"Score D (percentile)\", \"geometry\"]\n",
+    "    ].reset_index(drop=True)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "qAAEr1z-WZAT"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_simplified.rename(\n",
+    "        columns={\"Score D (percentile)\": \"D_SCORE\"}, inplace=True\n",
+    "    )"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "SCNUjEbzWg-o"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_cbg_compressed = gpd.GeoDataFrame(\n",
+    "        usa_simplified, columns=[\"D_SCORE\", \"geometry\"], crs=\"EPSG:4326\"\n",
+    "    )"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "Ej70uX0AmW0J",
+    "outputId": "88908f5e-b62d-494f-f0ea-649089b6652a"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_cbg_compressed.to_file(\"usa_cbg_scoreD.geojson\", driver=\"GeoJSON\")"
```

Comment:
Same question here, why is this saved?

Reply:
like the above, the thought is we might want to test whether national tiles that only have one property are performant